herepy/routing_api.py

```diff
@@ -191,6 +253,71 @@ def truck_route(self,
             modes = [RouteMode.truck, RouteMode.fastest]
         return self._route(waypoint_a, waypoint_b, modes)
 
+    @staticmethod
+    def _get_route_from_non_vehicle_maneuver(maneuver):
+        """Extract a short route description from the maneuver instructions."""
+        road_names = []
+
+        for step in maneuver:
+            instruction = step["instruction"]
+            try:
+                road_name = instruction.split('<span class="next-street">')[1].split(
+                    "</span>"
+                )[0]
+                road_name = road_name.replace("(", "").replace(")", "")
+
+                # Only add if it does not repeat
+                if not road_names or road_names[-1] != road_name:
+                    road_names.append(road_name)
+            except IndexError:
+                pass  # No street name found in this maneuver step
+        route = "; ".join(list(map(str, road_names)))
+        return route
+
+    @staticmethod
+    def _get_route_from_public_transport_line(
```

Comment:
I guess you use new line for the parameter not to exceed the max line length?

Reply:
I copied them over from a component which was blackened (https://github.com/psf/black) and forgot to adjust the formatting.

Do you just want to merge it or should I try to change the formatting?

Reply:
No need to reformat I think it's fine.